easybuild/easyconfigs/s/supermagic/supermagic-20170824-foss-2017a.eb

```diff
@@ -0,0 +1,32 @@
+##
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'supermagic'
+version = '20170824'
+
+homepage = 'https://github.com/hpc/supermagic'
+description = """Very simple MPI sanity code. Nothing more, nothing less."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+git_commit_id = '07311ad853'
```

Comment:
@jordiblasco Please move this line up right below the `version =` line, to avoid that it is overlooked when updating to a new version.